Add ABP configuration to TTI devices

Jaime-Trinidad opened the pull request:
<!--
Thanks for submitting a pull request. Please fill the template below,
otherwise we will not be able to process this pull request.
-->

#### Summary
<!--
A short summary, referencing related issues:
Closes #0000, References #0000, etc.
-->

Add ABP configuration to avoid break routing when using ABP for EU868 #110 

#### Changes
<!-- What are the changes made in this pull request? -->

- ...
- ...

#### Notes for Reviewers
<!--
NOTE: This section is optional.

Motivate briefly why it is implemented this way, if that deviates from the
implementation proposal in the referenced issues.
- How should your reviewers approach this pull request?
- @mention reviewers with special requests or questions for them
-->

...

#### Release Notes
<!--
NOTE: This section is optional.

Any notes that we need to include in the Release Notes for the next release.
These notes are formatted as bullet points, written in past tense, and will be
combined with the labels of this Pull Request.

Always mention changes in API.
-->

- ...

htdvisser commented on vendor/the-things-products/the-things-uno-abp.yaml:
```diff
@@ -0,0 +1,29 @@
+name: The Things Uno ABP
```
Shouldn't this just be a different profile for the existing device?

htdvisser reviewed (commented):
I'm not the expert on these devices (otherwise I'd have done it myself), but this doesn't look quite right to me yet.

htdvisser commented:
@johanstokking since you contributed quite a lot to these Arduino SDKs I think it's better if you reviewed this.

johanstokking reviewed (changes requested):
Agreed with @htdvisser's points:

1. There is no ABP example for The Things Node
2. The Things Uno has an ABP example (`SendABP`: https://github.com/TheThingsNetwork/arduino-device-lib/blob/ea60c59a726d4147aad878a928da5c8baa7faa50/examples/SendABP/SendABP.ino), and that should simply be a firmware version with a profile. This should be under the `the-things-uno` end device
3. RX1 delay is 1 second

Jaime-Trinidad commented:
I fixed files:

1. I delete ABP on Generic Node.
2. Added a firmware version with a profile in The Things Uno file.
3. Fixed RX1 delay to 1 second.

johanstokking commented on vendor/the-things-products/the-things-uno.yaml:
```diff
@@ -2,6 +2,10 @@ name: The Things Uno
 hardwareVersions:
   - version: '1.0'
+    numeric: 1
+  - version: '1.0-ABP'
+    numeric: 2
```
The hardware is the same

Jaime-Trinidad replied:
ok, now is fixed, same hardware version
